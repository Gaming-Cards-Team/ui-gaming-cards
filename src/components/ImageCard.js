import React from "react";
import Card from 'react-bootstrap/Card'

const ImageCard = (props) => {
  const { image } = props;
  return (  
    <Card.Img variant="top" src={image} />
  );
};

export default ImageCard;
