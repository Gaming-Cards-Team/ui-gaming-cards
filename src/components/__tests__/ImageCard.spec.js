import React from "react";
import ImageCard from "../ImageCard";
import renderer from "react-test-renderer";

describe("Image Card component", () => {
  it("should render successfully", () => {
    const component = renderer.create(<ImageCard />).toJSON();

    expect(component).toMatchSnapshot();
  });
});