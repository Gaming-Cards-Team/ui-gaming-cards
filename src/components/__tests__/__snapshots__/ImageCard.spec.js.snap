// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image Card component should render successfully 1`] = `
<img
  className="card-img-top"
/>
`;
